Makefile

```diff
@@ -282,6 +282,11 @@ else
 	npx react-native run-ios
 endif
 
+# Get all clojure files, including untracked, excluding removed
+define all_clojure_files
```

Comment:
Can you move this below into the `Tests` section?

Reply:
Also, this is a function, so the name should probably be `find_all_clojure_files`.